framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -330,6 +333,106 @@ public void processClassTree(ClassTree classTree) {
         super.visitClass(classTree, null);
     }
 
+    /**
+     * Check that the field invariant meets the following requirements:
+     *
+     * <p>1.) If the superclass of {@code classTree} has a field invariant, then the field invariant
```

Comment:
You can specify numbers in `<li>` items.  But even if you retain the current formatting, please add a `<!-- ... -->` comment telling future maintainers to update the body if they update the list.  Otherwise it will be too easy for someone editing this comment not to realize the damage they are doing.